.github/scripts/clean-cross.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e # exit on error
```

Comment:
It is reused in many places, it makes sense to move it to a shared config

Reply:
Moved to `config.sh`.